pro/src/components/pages/Offerers/Offerer/VenueV1/VenueCreation/VenueCreation.jsx

```diff
@@ -200,6 +227,14 @@ const VenueCreation = () => {
         <AccessibilityFields />
         <WithdrawalDetailsFields isCreatedEntity readOnly={readOnly} />
         <ContactInfosFields readOnly={false} />
+        {enableAdageVenueInformation && (
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** Si tu met cette condition ici, est ce qu'on a besoin de s'embêter à faire 
```js
const canOffererCreateCollectiveOfferRequest = enableAdageVenueInformation
        ? canOffererCreateCollectiveOfferAdapter(offererId).then(
            ({ payload }) => payload.isOffererEligibleToEducationalOffer
          )
        : Promise.resolve(false)
```
au dessus ?
On pourrait directement faire 
```js
const canOffererCreateCollectiveOfferRequest = canOffererCreateCollectiveOfferAdapter(offererId).then(
  ({ payload }) => payload.isOffererEligibleToEducationalOffer
)
```
avec le même affichage
‏

Reply:
C'est pour ne pas faire la requête tant qu'on a pas le FF activé :)